backends/cuda/runtime/cuda_backend.cpp

```diff
@@ -83,6 +83,12 @@ class ET_EXPERIMENTAL CudaBackend final
 
     LOAD_SYMBOL(handle, run, AOTInductorModelContainerRun, so_handle);
 
+    LOAD_SYMBOL(
+        handle,
+        update_constants_from_blob,
+        AOTInductorModelUpdateConstantsFromBlob,
+        so_handle);
```

Comment:
This is not backwards compatible

LOAD_SYMBOL fails if the symbol doesn't exist. 

We sometimes need to load the old style .so files -- with the weight embedded into so

Reply:
My understanding is this will always be there? @yushangdi can you confirm?

Reply:
I guess we won't have it before this API is added to pytorch/pytorch that's true.

Reply:
@larryliu0820 - i guess my question is whether there will be .so file only without the weight_blob